docs/native-tokens/minting-nfts.md

```diff
@@ -388,7 +388,7 @@ As with every other transaction, we need to calculate the fee and the output and
 
 Use this command to set the <i>$fee</i> variable.
 ```bash
-fee=$(cardano-cli transaction calculate-min-fee --tx-body-file matx.raw --tx-in-count 1 --tx-out-count 1 --witness-count 1 --mainnet --protocol-params-file protocol.json | cut -d " " -f1)
+fee=$(cardano-cli transaction calculate-min-fee --tx-body-file matx.raw --tx-in-count 1 --tx-out-count 1 --witness-count 2 --mainnet --protocol-params-file protocol.json | cut -d " " -f1)
```

Comment:
2 is correct for the `--witness-count` parameter when minting a token. However, `--tx-in-count` and `--tx-out-count` can be 0 I believe. In the code, they are used to add additional bytes to the fee calculation, but those bytes are actually already captured inside the matx.raw file. So, making those values 1 just makes you pay extra fees.  As long as you set something like 300000 for the fee when building the dummy transaction, setting `--tx-in-count` and `--tx-out-count` to 0 should work.